gists/pull-request-guidelines.md

```diff
@@ -0,0 +1,64 @@
+# Pull Request (PR) Guidelines
+
+Guidelines for creating pull requests at LaunchPad Lab.
+
+## Authors
+- Use a [pull request template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)
```

Comment:
I think this can be omitted, or replaced with "if there's a PR template follow the steps" (as-written it sounds like you need to add one yourself)